File: amazone_teckathon/lib/Models/Charity.dart


import 'package:amazone_teckathon/Models/Event.dart';
import 'package:amazone_teckathon/Models/Requist.dart';

class Charity {
  String email;
  String password;
  String name;
  String discription;
  var logo;
  List<Request> charitydonations;
  List<Event> events;
  String accountnumber;

  Charity(
      {this.discription,
        this.accountnumber,
      this.charitydonations,
      this.email,
      this.events,
      this.logo,
      this.name,
      this.password});

  factory Charity.fromJson(Map<String, dynamic> json) =>
      _$productFromJson(json);
}

Charity _$productFromJson(Map<String, dynamic> json) {
  return Charity(
      discription: json['discreption'].toString(),
      charitydonations: json['charityDonations'],
      email: json['email'],
      events: json['events'],
      logo: json['logo'],
      name: json['name'],
      accountnumber: json["bankAccount"],
      password: json['password']);
}

File: TAlaApi/TAlaApi/Models/Charity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TAlaApi.Models
{
    public class Charity
    {
        public int CharityID { get; set; }

        public string Email { get; set; }

        public string Password { get; set; }

        public string Name { get; set; }

        public string Discreption { get; set; }

        public Byte[] img { get; set; }

        public List<Request> CharityDonations { get; set; }

        public List<Event> Events { get; set; }
    }
}
